package.json

```diff
@@ -32,6 +32,7 @@
         "font-awesome": "4.7.0",
         "lodash": "4.17.21",
         "lodash.product": "^18.10.0",
+        "moment": "^2.30.1",
```

Comment:
isnt moment already available? its used in the code already, this might not be neccessary

Reply:
Yes.. I just checked and it is available, I don't know how though. Anyways I think it's better to have it in package.json either way, so it's clear for someone new that we use that package. What do you think?